packages/neutrino/src/neutrino.js

```diff
@@ -23,8 +23,44 @@ class Neutrino extends EventEmitter {
     this.options = merge(options, { root, source, output, tests, node_modules, entry });
   }
 
-  use(preset, options = {}) {
-    preset(this, options);
+  use(middleware, options = {}) {
+    middleware(this, options);
+  }
+
+  import (middlewares) {
+    middlewares = Array.isArray(middlewares) ? middlewares : [middlewares];
+    this.requireMiddlewares(middlewares).forEach(middleware => api.use(preset));
+  }
+
+  requireMiddlewares(middlewares) {
```

Comment:
We can remove this method, if we do the mapping in `requireMiddleware`, and normalize the array. See my comment there.